lib/python/treadmill_aws/ec2client.py

```diff
@@ -189,6 +206,169 @@ def get_vpc_id_by_tags(ec2_conn, tags):
     return get_vpc_by_tags(ec2_conn, tags)['VpcId']
 
 
+def create_image(
+        ec2_conn, image_name, cloud_init, base_image_id,
+        image_instance_type, image_instance_key, image_instance_role,
+        image_instance_secgroup_ids, image_subnet_id, image_version
+):
+    # pylint: disable=R0915
```

Comment:
so this whole method is not what we want really.

the flow is:
 - create_instance, and use unique generated name, pass it as user data to the instance.
 - while applying changes, the instance will trigger "snapshot self". as part of the snapshot, it will use the uniq name as new image name. 
 - the api returns the uniq id, passes it to cli, and it is passed to the user. 

everything is async. from then on, something else will poll for this image name to materialize, and it will tag it appropriately for future use. so this entire function is gone and needs to be rewritten.